libs/elements/layout/texts/src/lib/elewa-text-content-item/elewa-text-content-item.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="text-card" *ngFor="let item of Content ">
+    <h2>{{item.title}}</h2>
+    <ul>
+        <li *ngFor="let description of item.descriptions">{{description}}</li>
+    </ul>
+</div>
```

Comment:
Our component is a reusable component, which means it only renders one item. The mock data will be passed from the parent component to this that will be created in another issue